### PR TITLE
Docs: Added Alert list visualization

### PR DESCRIPTION
**Pull request by oddlittlebird**
Please check this carefully. All the Filter and State field documentation is new, and I expanded some of the existing Options field documentation.

### CODE REVIEW

File: docs/sources/panels/visualizations/alert-list-panel.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+Use these settings to refine your visualization.
+
+### Options
+
+- **Show -** Choose whether the panel should display the current alert state or recent alert state changes.
+- **Max Items -** Sets the maximum number of alerts to list.
+- **Sort order -** Select how to order the alerts displayed:
+  - **Alphabetical (asc) -** Alphabetical order.
+  - **Alphabetical (desc) -** Reverse alphabetical order.
+  - **Importance -** By importance.
```
Comment (daniellee) [inline docs/sources/panels/visualizations/alert-list-panel.md:31]: There is a ranking that is used to sort by importance:

- alerting: 1,
- no_data: 2,
- pending: 3,
- ok: 4,
- paused: 5

Maybe can mention that?

**Review by daniellee — APPROVED** [review]
LGTM - one possible addition to explain what importance is when sorting.

File: docs/sources/panels/visualizations/alert-list-panel.md (hunk 1 above)
Comment (dprokop) [reply docs/sources/panels/visualizations/alert-list-panel.md:31]: Yes, I think this is a good idea.

**Review by dprokop — APPROVED** [review]

File: docs/sources/panels/visualizations/alert-list-panel.md (hunk 1 above)
Comment (oddlittlebird) [reply docs/sources/panels/visualizations/alert-list-panel.md:31]: Will do

**Pull request merged**